src/schema/index.ts

```diff
@@ -4,11 +4,13 @@ const Schema = gql`
   type User {
     id: ID!
     role: String!
+    name: String
     email: String!
     password: String!
     profile: Profile!
   }
   input RegisterInput {
+    name: String
```

Comment:
Same here, this field is not recommended to have on user model.

Only fields related to `Authentication (email, password) and authorization (role, active state)` should be on User model. All others are on profile